### ISSUE TITLE
Parameters set with same input

### ISSUE THREAD

**Issue by mhamilt** [kohnoguchi/cityscape-instrument#3]
# Parameters set with same input

If you are trying to set play back speed and amplitude with different inputs then `BandPass_with_File1` is not coded correctly. You will want to take a look at the `serialBuffer` array on [line 50](https://github.com/kohnoguchi/cityscape-instrument/blob/master/Final%20Crit/BandPass_with_File1/BandPass_with_File1.pde#L50) and an example of how you can use it on [line 59](https://github.com/kohnoguchi/cityscape-instrument/blob/master/Final%20Crit/BandPass_with_File1/BandPass_with_File1.pde#L59)

Basically, you shouldn't be using `val` as it is just a single number and you will need to access multiple values in succession.
